src/implementations/twig/components/notification/notification.html.twig

```diff
@@ -18,7 +18,7 @@
 
 {# Internal properties #}
 
-{% set _variant = variant|default('') %}
+{% set _variant = variant|default('info') %}
```

Comment:
this is good, but at the sametime it might impact some implementations, who knows if someone used this component without setting any variant, indeed...
Anyhow i think that it's the right thing to do.